docs/blog/2018-06-07-build-a-gatsby-blog-using-the-cosmic-js-source-plugin/index.md

```diff
@@ -0,0 +1,424 @@
+---
+title: Build a Gatsby Blog using the Cosmic JS source plugin
+date: "2018-06-07"
+author: "Tony Spiro"
```

Comment:
Can you add `canonicalLink` here to your original post so you get to keep the google juice ;-) https://github.com/gatsbyjs/gatsby/blob/82d9188aba871bb0bb94e32c3e590cb13ae3cd12/docs/blog/2018-04-10-how-to-handle-comments-in-gatsby-blogs/index.md#L5

Reply:
Thank you @KyleAMathews! This is done.